contrib/epee/include/net/abstract_tcp_server2.inl

```diff
@@ -628,7 +630,17 @@ PRAGMA_WARNING_DISABLE_VS(4355)
             return false; // aborted
         }*/
 
-        long int ms = 250 + (rand()%50);
+        using engine = std::mt19937;
```

Comment:
Mersenne twister is good, but it's not a cryptographically secure PRNG. Is this bug description actually correct?
This is just generating a random backoff before attempting to send more data on a socket. The actual sleep value used here is kind of irrelevant, since the kernel level socket buffers will obscure whatever exact delay value was used.

I think Coverity is just being its usual brain-damaged self here.